packages/desktop-client/src/components/mobile/budget/ExpenseCategoryListItem.tsx

```
@@ -37,19 +42,35 @@
     isSidebar: true,
   });
+  const { schedule, status: scheduleStatus } = useCategoryScheduleGoalTemplate({
+    category,
+  });
+  const isScheduleUpcomingOrMissed =
+    scheduleStatus === 'upcoming' ||
+    scheduleStatus === 'due' ||
+    scheduleStatus === 'missed';
+
+  const isScheduleRecurring =
+    schedule && schedule._date && !!schedule._date.frequency;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)